### PR TITLE
Closes #16. First user created is an admin by default.

### LINKED ISSUE #16: Make the first user that signs in the admin, not "justin@bitovi.com".
Currently, the only user marked `isAdmin` is a user with an email address of `justin@bitovi.com` or `justinbmeyer@gmail.com`.  This happens here: https://github.com/donejs/bitballs/blob/master/services/users.js#L26

To close this issue:

- [ ] - Change the service layer to check the number of users.  If there are 0 users, that user is the admin.  If there are more than one user, the user is not the admin. You can use [Bookshelf's count](http://bookshelfjs.org/#Model-static-count) to check how many users there are.

### PR DESCRIPTION
**Pull request by James0x57**
Also updated gitignore for the DS_Store and log files.
Also added req.isDev that will be useful later but probably won't stay in later.

Closes #16

### CODE REVIEW

File: services/users.js [hunk 1]
```diff
@@ -52,6 +52,20 @@ app.post('/services/users', passport.authenticate('signup'), function(req, res)
...
+		//new User({id:3}).destroy();
+		//new User({id:4}).destroy();
+		User.fetchAll().then( function ( list ) {
+			res.send( list );
+		});
+	} else {
+		res.status( 500 ).send( "Only available in development mode." );
+	}
+});
+
```
Comment (akagomez) [inline services/users.js]: Why is this necessary?

File: public/components/user/view-model.js [hunk 2]
```diff
@@ -47,4 +47,11 @@ module.exports = can.Map.extend({
 			}
 		});
 	}
+	//, getList: function () {
+	//	return User.getList();
+	//},
+	//init: function () {
+	//	window.userVM = this;
+	//	//userVM.getList();
+	//}
```
Comment (akagomez) [inline public/components/user/view-model.js]: Was this meant to be committed? If so, I think it should be deleted.

File: services/users.js (hunk 1 above)
Comment (James0x57) [inline services/users.js]: It gives us a place to see the list of users and their properties while we're developing. I had to add it to check if the first user was an admin, and then to figure out why it wasn't. I wrapped it in the isDev thing afterwords which we won't be able to use as is until we have the 'donejs develop' thing added back with issue 12 which should be done earlyish this week.

File: public/components/user/view-model.js (hunk 2 above)
Comment (James0x57) [inline public/components/user/view-model.js]: I deleted it then put it back - wasn't sure if it would be useful later. Will just delete things like this in the future for this project. Will push with it removed in a sec.

File: services/users.js (hunk 1 above)
Comment (akagomez) [inline services/users.js]: I would rather this sort of debugging should be done in a postgres client.

**Comment by akagomez** [comment]
+1

**Comment by akagomez** [comment]
Can you rebase this into one commit?
